src/OneBlinkUploader.ts

```diff
@@ -246,6 +247,7 @@ export default class OneBlinkUploader {
         lastElementUpdated,
         externalId,
         device,
+        submissionTimestamp,
```

Comment:
```suggestion
        submissionTimestamp: new Date().toISOString(),
```

No need to pass this in.